apps/src/simpleSignUp/lti/iframe/LtiIframePage.tsx

```diff
@@ -14,12 +14,25 @@ interface LtiIframePageProps {
 export const LtiIframePage = ({logoUrl, authUrl}: LtiIframePageProps) => {
   const [callToActionDisabled, setCallToActionDisabled] =
     useState<boolean>(false);
+  const [textContent, setTextContent] = useState(i18n.ltiIframeDescription());
 
   const handleCallToAction = () => {
     window.open(authUrl, '_blank');
     setCallToActionDisabled(true);
+    setTextContent(i18n.ltiIframeRefresh());
   };
 
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      if (!callToActionDisabled) {
+        setCallToActionDisabled(true);
+        setTextContent(i18n.ltiIframeTimedOut());
+      }
+    }, 300000); // Time out after 5 minutes
```

Comment:
```suggestion
    }, IFRAME_TIMEOUT_MILLISECONDS);
```
Could we pull out the number to a constant to avoid [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming))?

Reply:
Also would we be able to add a unit test for this? We should be able to use the [jest timers api](https://jestjs.io/docs/timer-mocks#advance-timers-by-time) for this.

Reply:
Yes, I will look into that!